rasa/core/policies/ensemble.py

```diff
@@ -27,22 +27,17 @@
     ACTION_BACK_NAME,
 )
 from rasa.shared.core.domain import InvalidDomain, Domain
-from rasa.shared.core.events import (
-    SlotSet,
-    ActionExecuted,
-    ActionExecutionRejected,
-    Event,
-)
+from rasa.shared.core.events import ActionExecutionRejected
 from rasa.core.exceptions import UnsupportedDialogueModelError
 from rasa.core.featurizers.tracker_featurizers import MaxHistoryTrackerFeaturizer
 from rasa.shared.nlu.interpreter import NaturalLanguageInterpreter, RegexInterpreter
 from rasa.core.policies.policy import Policy, SupportedData
 from rasa.core.policies.fallback import FallbackPolicy
 from rasa.core.policies.memoization import MemoizationPolicy, AugmentedMemoizationPolicy
 from rasa.core.policies.rule_policy import RulePolicy
-from rasa.shared.core.trackers import DialogueStateTracker
+from rasa.shared.core.trackers import DialogueStateTracker, create_action_fingerprints
```

Comment:
please refer to functions by their modules

Reply:
I tried, it becomes impossibly long 🙈
this is not ambiguous function, no?

Reply:
it's our Python Code convention to refer to functions using their module so there is no way around it.